src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,39 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
 import core.basesyntax.model.User;
+import java.util.Objects;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    public static final int MIN_AGE = 18;
+    public static final int MIN_PASSWORD_CHARACTERS_AMOUNT = 6;
```

Comment:
```suggestion
    public static final int MIN_PASSWORD_CHARACTERS_AMOUNT = 6;
    private StorageDao storageDao = new StorageDaoImpl();
```

давай вынесем переменную storageDao на уровень класса